libs/packages/sam-formly/src/lib/formly-filters/advanced-filters/advanced-filters.component.ts

```diff
@@ -152,7 +165,61 @@ export class AdvancedFiltersComponent {
         ]
       }
     ];
-    const updateField: FormlyFieldConfig[] = [...filedGroup, ...modalFields];
+
+    const showInactiveGroup: FormlyFieldConfig[] = [
```

Comment:
When we discussed this, I thought we were making this configurable so developers would pass in the formly configuration they wanted. 

I also think we should make it more generic where they can pass in things that come before and after the hide filters configuration. 

```
[
   ...preFormlyFieldConfig,  //input
   ...toggleHiddenFieldsFormlyFieldConfig, 
   ...postFormlyFieldConfig  //input
]
```

Reply:
I thought we decided against making it more configurable than it already was. They can still pass in the configuration they want, we are just inserting into our structure which includes selectAll, and potentially inactiveFilterValues.

If we want to open it up more, I can take a look at that in sprint 3. I'll just need a bit of time with you to discuss what you're wanting the resulting config to look like.

Reply:
Followed up with Dave on this, will go ahead and work with Nisha to open a backlog item to enhance in this manner.